package.json

```diff
@@ -60,7 +60,7 @@
     "npm": "^2.14.0"
   },
   "scripts": {
-    "build": "gulp",
+    "build": "git clone -b fix\/tracker-ebola git://github.com/frictionlessdata/jsontableschema-js.git node_modules/jsontableschema && cd node_modules/jsontableschema && npm install && npm run build && gulp",
```

Comment:
I'm not sure how it works? `npm run build` calls `npm run build`?

Could this close logic be moved to `.travis.yml`?